example.py

```diff
@@ -156,22 +203,39 @@ def test_run_a_remote_command_as_root():
         assert result["succeeded"]
 
 
+def test_run_a_remote_command_in_a_different_dir():
+    "run a simple `remote` command in a different remote directory"
+    with remote_env() as env:
+        with test_settings():
+            temp_dir = env["temp-dir"]
+            with rcd(temp_dir):
+                result = remote("pwd")
+    assert result["succeeded"]
+    assert temp_dir == first(result["stdout"])
```

Comment:
wrap in a list rather pull first result out. `first` implies there are more than one result, which there isn't and shouldn't be.